java/arcs/core/data/expression/Evaluator.kt

```diff
@@ -164,42 +168,69 @@ sealed class GlobalFunction(val name: String) {
       toSequence<Any>(args[0]).asIterable().union(
         toSequence<Any>(args[1]).asIterable()
       ).asSequence()
+
+    override fun inferredType(evaluator: TypeEvaluator, args: List<InferredType>) =
```

Comment:
Can this fit on one line? If not, then it should be wrapped with `{}` (according to the recent Kt tech talk).

Reply:
This style, if accepted, should apply to the rest of the file.